src/core/server/ui_settings/saved_objects/migrations.ts

```diff
@@ -26,6 +26,28 @@ export const migrations = {
         {}
       ),
     }),
+    ...(doc.attributes && {
```

Comment:
I think this new spread is removing any changes applied in the spread coming from lines 14 to 28. I'd change the logic above to look like:
```
if (siemUseCase) {...} 
else if (timepickerQuickRangesMigrations) { ... } 
else { ... }
```

Reply:
@afharo done